### PR TITLE
Fix velocities for Steering Chassis

### PR DESCRIPTION
**Pull request by ywh114**
Normalize velocities from square control pad
Add velocity limit option
Add safety check when turning and control max delta v

### CODE REVIEW

**Comment by ywh114** [comment]
just look at last commit

File: vehicles/Steering/chassis.cc [hunk 1]
```diff
@@ -206,14 +207,60 @@ void chassisTask(void* arg) {
...
+#ifdef LIMITED_SAMPLE_MATCH
+  v_max *= 0.5;
+#endif
+  // Prevent vehicle from flipping due to sudden changes in velocity.
+  // Speed at which safety will not kick in
+  // TODO: Find good numbers
+  float momentum_safe_speed = v_max * 0.4; // Magic number
+  float momentum_safe_factor_line = 0.3;   // also ^
+  float momentum_safe_factor_turn = 0.3;   // also ^
+  float momentum_factor = 0.75;            // also ^
```
Comment (AzulRadio) [inline vehicles/Steering/chassis.cc:221]: Please use uppercase for const name. Also, consider using `constexpr float` or `const float`

File: vehicles/Steering/chassis.cc [hunk 2]
```diff
@@ -206,14 +207,60 @@ void chassisTask(void* arg) {
   chassis->SteerThetaReset();
   chassis->SetWheelSpeed(0,0,0,0);
 
+  float v_max = 660;
+  float normalization_factor = 1;
+#ifdef LIMITED_SAMPLE_MATCH
+  v_max *= 0.5;
```
Comment (AzulRadio) [inline vehicles/Steering/chassis.cc:213]: Same here, see comment below.

**Review by AzulRadio — CHANGES_REQUESTED** [review]
Minor changes requested. Other parts LGTM. Need to test on the robot.

**Review by AzulRadio — APPROVED** [review]
LGTM

**Pull request merged**